www/index.js

```diff
@@ -739,10 +742,29 @@ function init() {
     show("#details_custom");
   }
 
-  setupSelectList($("#versions"), Object.keys(config.versions), (version) => {
+  let versions = await fetch(config.image_url + "/.versions.json", { cache: "no-cache" })
+    .then((obj) => { return obj.json(); })
+    .then((obj) => { return obj.versions_list.filter((version) => version.localeCompare("19.07.4") >= 0);});
+
+  if (config.show_snapshots) {
+    let branches = [];
+    for (const version of versions.slice()) {
+      let branch = version.split('.').slice(0, -1).join('.') + '-SNAPSHOT';
```

Comment:
I do not like to hard code this OpenWrt specific stuff. I wonder if this can be done in a nicer way.
Shouldn't we but this in the .versions.json file?

Reply:
Sure would would you add it?